contracts/Registries/Overrides.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.11;
+
+import "../Utilities/Ownable.sol";
+
+interface ICurveRegistry {
+    function get_pool_from_lp_token(address) external view returns (address);
+
+    function pool_list(uint256) external view returns (address);
+
+    function pool_count() external view returns (uint256);
+}
+
+interface ICurveAddressesProvider {
+    function max_id() external view returns (uint256);
+
+    function get_address(uint256) external view returns (address);
+}
+
+contract CurveRegistryOverrides is Ownable {
+    ICurveAddressesProvider public curveAddressesProvider;
+    ICurveRegistry public curveRegistry;
```

Comment:
not sure if you already know this, but you can get the registry from the address provider

Reply:
Updated to use addresses provider to retrieve registries.